GitHub_Learning/Feature/Onboarding/Application/LoadOnboardingFinishedFacade.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  LoadIsFinishedFacade.swift
+//  GitHub_Learning
+//
+//  Created by Amin Ashhab on 24.11.2020.
+//  Copyright © 2020 Amin_Second_Test_Project. All rights reserved.
+//
+
+import Foundation
+import CleanCore
+import CleanPlatform
+
+protocol LoadOnboardingFinishedFacade {
+    func load(_ request: LoadOnboardingFinishedRequest, completion: @escaping ((Result<LoadOnboardingFinishedResponse>) -> Void))
```

Comment:
Musi tu byt completion? Nemuze ti to rovnou vratit result?

Reply:
Ten jsem tam měl původně. Řešil to právě nějakou dobu, je tam malá časová prodleva, a ještě když to volám při startu aplikace, tak jsem nepříšel na to jak to jinak udělat. A DyspatchQueue.after nebo Timer jsem tam nechtěl dávat. Tak jsem tam dal completion no. Nenapadá mě jak jinak asi no.

Reply:
Podivame se na to. Zajima me, kde je ta prodleva.